app/src/main/res/values/strings.xml

```diff
@@ -82,6 +86,10 @@
     <string name="menu_app_remove">Remove from history</string>
     <string name="menu_app_details">Details</string>
     <string name="menu_app_uninstall">Uninstall</string>
+    <string name="menu_app_hibernate">Hibernate</string>
+    
+    <string name="toast_hibernate_completed">hibernated successfully.</string>
```

Comment:
use String.format and %s? This helps for translating in langges where the placeholder may be at the end of the sentence